### PR TITLE
Added my name

### PR DESCRIPTION
**Pull request by MDGcode**

### CODE REVIEW

**Comment by lseIT** [comment]
foloseste priettier :)

**Comment by cristim67** [comment]
foloseste priettier :)

File: client/src/App.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ function App() {
         >
           Learn React
         </a>
+        <h1>Adi</h1>
```
Comment (cristim67) [inline client/src/App.js]: Nume complet

**Comment by cristim67** [comment]
good job :)

**Pull request merged**